assets/app/scripts/controllers/javaLink.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+/**
+ * @ngdoc function
+ * @name openshiftConsole.controller:JavaLinkController
+ * @description
+ * # ProjectController
+ * Controller of the openshiftConsole
+ */
+angular.module('openshiftConsole')
```

Comment:
Since this isn't providing a controller, let's put it under a new folder called "extensions" or something... would like to put it under a different module name as well if possible to keep dependencies as cleanly separated as possible